workspace.json

```diff
@@ -4604,7 +4604,8 @@
             "passWithNoTests": true
           }
         }
-      }
+      },
+      "docker-next": {}
```

Comment:
I think you need `docker-express` rather. This is a NestJS app, not a NextJS one, right?

Reply:
haha yeah, the frontend is a next app :P
My bad